components/FilterControls.tsx

```diff
@@ -86,7 +86,7 @@ export function FilterControls({
           unresolved: !extraFilters.resolvingSoon,
         }),
       filterActive: extraFilters.resolvingSoon,
-      className: "@xl:block",
+      className: "hidden",
```

Comment:
It's better/cleaner to hide these at all resolutions I think. Before/after these changes at 620px:
<img width="616" alt="Screenshot 2024-09-25 at 12 32 23" src="https://github.com/user-attachments/assets/b2abd618-522a-4fb5-80ed-1a3fe5c9b7be">
<img width="616" alt="Screenshot 2024-09-25 at 12 33 15" src="https://github.com/user-attachments/assets/0774c173-50a2-4c40-b466-a4a15fa9d5d3">

Reply:
Agree!